EventFilter/GEMRawToDigi/interface/AMCdata.h

```diff
@@ -8,54 +8,63 @@ namespace gem {
-      uint64_t l1AID : 24;       // L1A number – basically this is like event number, but it’s reset by resync
+      uint64_t l1AID : 24;       // L1A number – basically this is like event number, but reset by resync
       uint64_t AMCnum : 4;       // Slot number of the AMC
       uint64_t reserved : 4;     // not used
     };
   };
   union AMCheader2 {
     uint64_t word;
     struct {
-      uint64_t boardID : 16;  // 8bit long GLIB serial number
-      uint64_t orbitNum : 16;
-      uint64_t param3 : 8;
-      uint64_t param2 : 8;
-      uint64_t param1 : 8;
+      uint64_t boardID : 16;   // 8bit long GLIB serial number
+      uint64_t orbitNum : 16;  // Orbit number, Reset by EC0
+      uint64_t param3 : 8;     // RunPar - Controlled by software, normally used only for calibrations
+      uint64_t param2 : 8;     // RunPar - Controlled by software, normally used only for calibrations
+      uint64_t param1 : 8;     // RunPar - Controlled by software, normally used only for calibrations
       uint64_t runType : 4;    // run types like physics, cosmics, threshold scan, latency scan, etc..
       uint64_t formatVer : 4;  // Current format version = 0x0
     };
   };
   union AMCTrailer {
     uint64_t word;
     struct {
-      uint64_t dataLengthT : 20;  // Overall size of this FED event fragment
-      uint64_t l1AIDT : 12;       // 8bit long GLIB serial number (first 8 bits)
-      uint64_t crc : 32;
+      uint64_t dataLength : 20;  // Number of 64bit words in this event
+      uint64_t l1AIDT : 12;      // 8bit long GLIB serial number (first 8 bits)
+      uint64_t crc : 32;         // CRC added by the AMC13
     };
   };
   union EventHeader {
     uint64_t word;
     struct {
-      uint64_t ttsState : 11;   // GLIB TTS state at the moment when this event was built.
-      uint64_t davCnt : 5;      // Number of chamber blocks
-      uint64_t buffState : 24;  // buffer error
-      uint64_t davList : 24;    // Bitmask indicating which inputs/chambers have data
+      uint64_t ttsState : 4;  // GLIB TTS state at the moment when this event was built.
+      uint64_t
+          pType : 4;  // Payload type: can be one refering to different zero suppression schemes in normal data taking or calibration type. Note in calibration type the entire GCT (GEM Chamber Trailer) is skipped
+      uint64_t pVer : 3;        // Version of the payload type
+      uint64_t davCnt : 5;      // Number of chamber blocks in this event
+      uint64_t buffState : 24;  // Buffer status, Always 0 in current fw
+      uint64_t davList : 24;    // Data Available list: a bitmask indicating which chambers have data in this event
     };
   };
   union EventTrailer {
     uint64_t word;
     struct {
-      // GLIB is out‐of‐sync (critical): L1A ID is different for different chambers in this event (1 bit)
-      uint64_t oosGlib : 40;
-      uint64_t chTimeOut : 24;  // GLIB did not receive data from a particular input for this L1A
+      uint64_t
```

Comment:
If you split the comment into different lines you can avoid clang format splitting the declararation part